### ISSUE TITLE
Test/launch details in expandable section

### ISSUE THREAD

**Issue by slavamarkovski** [reportportal/reportportal#480]
Will be good to move launch details located under launch name (tags, description, owner etc.) in expandable section collapsed by default (or move default behavior in user properties). Similar for test entity (duration, tags, etc).

Th main idea is that all these properties can be requested on demand instead of having them always visible. and let them occupy useful space to have more launch/test record visible on the screen.

**Comment by evjlobanova** [comment]
@slavamarkovski  do you mean something like in this issue https://github.com/reportportal/reportportal/issues/453?

**Comment by slavamarkovski** [comment]
@evjlobanova nope, I mean that this information should be hidden by default in collapsible section as it takes a lot of useful place.

**Comment by evjlobanova** [comment]
@slavamarkovski I `ve got.
